tests/Hydration-test.js

```diff
@@ -0,0 +1,32 @@
+const chai = require('chai');
+const expect = chai.expect;
+
```

Comment:
I like how these classes are simple mirrors of each object in the data instance, I think that's what we're being guided toward.

Reply:
@srslie Agreed- I think that's the route to take for now!